app/pages/project/components/ProjectNavbar/components/ProjectTitle/ProjectTitle.jsx

```diff
@@ -48,12 +46,14 @@ export const StyledRedirect = styled.a`
 `;
 
 export const StyledCheckMarkWrapper = styled.span.attrs({
-  'aria-label': "Zooniverse Approved",
-  role: 'img',
-  title: "Zooniverse Approved"
+  role: 'img'
 })`
   font-size: 0.75rem;
-  margin-left: 1rem;
+  margin: 0 0 0 1rem;
+
+  .rtl & {
+    margin: 0 1rem 0 0;
+  }
 `;
```

Comment:
Could this also be fixed by adding a space in the HTML?

Reply:
Yeah this is a good candidate for just a space too.